angularjs-1.5-basics/src/app/landing-page/_landing-page.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+angular.module('landing-page', []);
```

Comment:
__[jshint]__
* 'angular' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->